docs/cart-and-checkout-blocks/extensibility-getting-started.md

```diff
@@ -0,0 +1,132 @@
+---
+post_title: Cart and Checkout - Extensibility getting started guide
+menu_title: Extensibility
+tags: reference
+---
+<!-- markdownlint-disable MD041 -->
+
+This document is a high-level overview of the moving parts required to extend the Cart and Checkout blocks.
+
+To get started, it is recommended to first read the [Block Development Environment](https://developer.wordpress.org/block-editor/getting-started/devenv/) documentation from WordPress and follow [Tutorial: Build your first block
+](https://developer.wordpress.org/block-editor/getting-started/tutorial/).
+
+## Example block template package
+
+There is an example block template in the WooCommerce repository. Having this template set up while reading this document may help to understand some of the concepts discussed. See the [`@woocommerce/extend-cart-checkout-block` package documentation](https://github.com/woocommerce/woocommerce/tree/trunk/packages/js/extend-cart-checkout-block#woocommerceextend-cart-checkout-block) for how to install and run the example block.
+
+(Note: the code in the repository linked above will not be very useful alone, the code there is templating code. It will be transformed into regular JS and PHP after following the README instructions.)
```

Comment:
```suggestion
(Note: the code in the repository linked above will not be very useful alone; the code there is templating code. It will be transformed into regular JS and PHP after following the README instructions.)
```